fatrace.c

```diff
@@ -251,6 +253,12 @@ print_event (const struct fanotify_event_metadata *data,
                 len--;
             procname[len] = '\0';
             procname_pid = data->pid;
+
+            /* get user and group */
+            if (option_user == 1) {
+                if (fstat (proc_fd, &ug) < 0)
+                    err (EXIT_FAILURE, "uid/gid stat");
```

Comment:
This seems a bit harsh -- this might be subject to races where the process already went away, and the  stat legitimately fails? Please do the same debug() as a failing procname read.

Reply:
Lowered to debug.